app/components/lead_providers/your_schools/table_row.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module LeadProviders
+  module YourSchools
+    class TableRow < BaseComponent
+      include PaginationHelper
+
+      with_collection_parameter :school
+
+      def initialize(school:, cohort:)
+        @school = school
+        @cohort = cohort
+      end
+
+    private
+
+      attr_reader :school, :cohort
```

Comment:
Are these attribute readers actually private?

And more importantly, do we want them to worry about this? I don't think any place ever attempts to do anything with view components other than render them, so having these attributes as public is not likely to break anything.

And I do like having this stuff at the top of the file, purely out of aesthetics.

Reply:
Private/public split is really quite philosophical. Making all the methods public won't break anything. Some companies even imposes a public-only policy (and some other work with JS where they have no other option). I personally like making a distinction between public and private API - this is however more for other developers than the requirement imposed by the code.

As per localisation of that call - from my experience it is just a matter of getting used to it. I absolutely hated that when encounter this style some time ago, it is pretty hard to write it differently now. :)